frontend/src/__tests__/integration/pages/projects/ProjectDetails.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { test, expect } from '@playwright/test';
+
+test('Empty project', async ({ page }) => {
+  await page.goto(
+    './iframe.html?args=&id=tests-integration-pages-projects-projectdetails--empty-details-page&viewMode=story',
```

Comment:
These items definitely need to be hidden away from our tests if this is needed...

Reply:
what do you suggest? this is how the storybook docs integrate with playwright. https://storybook.js.org/docs/react/writing-tests/importing-stories-in-tests#example-with-playwright

Reply:
`await goTo(page, 'something', 'other', 'than', 'this')` ... just easier... if there is complexity here to the path generated, maybe: `await goTo(page, { page: 'test', other: 'thing', ...)`

Is this generated by something that is just copy & paste?